cypress/Shared/MeasureGroupPage.ts

```diff
@@ -130,10 +129,10 @@ export class MeasureGroupPage {
     public static readonly denominatorObservation = '[id="measure-observation-denominator"]'
     public static readonly denominatorAggregateFunction = '[id="measure-observation-aggregate-denominator"]'
     public static readonly removeDenominatorObservation = '[data-testid="measure-observation-denominator-remove"]'
-    public static readonly measureObservationSelect = '[class="MuiMenuItem-root MuiMenuItem-gutters MuiButtonBase-root css-1km1ehz"]'
+    public static readonly measureObservationSelect = '[class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiPaper-root MuiMenu-paper MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation8 MuiPopover-paper css-177ic5c"]'
```

Comment:
is there a better name for this?

Reply:
Changed how I was accessing the observation values